docs/design/agent-workflows/wp-1-pi-tracing/poc/.env.example

```
@@ -0,0 +1,7 @@
+# Agenta collector (the runner also falls back to the repo-root .env.test.local).
+AGENTA_HOST=http://144.76.237.122:8280/
+AGENTA_API_KEY=your-agenta-project-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENTA_API_KEY key should go before the AGENTA_HOST key

(UnorderedKey)